.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+name: Test ros2_controllers
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+  schedule:
+    # Run every morning to detect flakiness and broken dependencies
+    - cron: '17 8 * * *'
+
+jobs:
+  build_and_test:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: ros-tooling/setup-ros@0.0.15
+      - uses: ros-tooling/action-ros-ci@master
+        with:
+          package-name: joint_state_controller joint_trajectory_controller
+          colcon-mixin-name: coverage-gcc
+          colcon-mixin-repository: https://raw.githubusercontent.com/colcon/colcon-mixin-repository/master/index.yaml
+          vcs-repo-file-url: |
+            ${{ github.workspace }}/travis/workspace.repos
```

Comment:
I still think it makes more sense to specify the ros2 master upstream repos here, followed by a dedicated one for ros2_controllers. 
I would further make this travis independent, as it might get decommissioned at some point.